src/main/resources/application.yml

```diff
@@ -1,6 +1,6 @@
 spring:
   profiles:
-    active: 'prod'
+    active: 'dev'
```

Comment:
@omgkanamikun, please take a look at file [README](https://github.com/bobocode-labs/gojohnny-rest-api#database). **You should not commit this change**